xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/XWiki/Lightbox/LightboxAdministration.xml

```diff
@@ -0,0 +1,332 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="XWiki.Lightbox.LightboxAdministration" locale="">
+  <web>XWiki.Lightbox</web>
+  <name>LightboxAdministration</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>XWiki.Admin</creator>
+  <creationDate>1639138843000</creationDate>
+  <parent>WebHome</parent>
+  <author>XWiki.Admin</author>
+  <contentAuthor>XWiki.Admin</contentAuthor>
+  <date>1640076427000</date>
+  <contentUpdateDate>1639244117000</contentUpdateDate>
+  <version>5.6</version>
+  <title>LightboxAdministration</title>
+  <comment/>
+  <minorEdit>true</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content>{{velocity output="false"}}
+#macro (stripHTMLMacro $displayOutput)
+  $stringtool.removeEnd($stringtool.removeStart($displayOutput, '{{html clean="false" wiki="false"}}'), '{{/html}}')
+#end
+#macro (showProperty $propertyName)
+  &lt;dl>
+    &lt;dt>
+      #set ($propertyClass = $xclass.get($propertyName))
+      &lt;label#if ($editing) for="${lightboxConfigClassName}_0_${propertyName}"#end>
+        $services.localization.render("lightbox.admin.${propertyName}.label")
+      &lt;/label>
+      #set ($hint = $propertyClass.hint)
+      #if ("$!hint" == '')
+        #set ($hint = $services.localization.render("lightbox.admin.${propertyName}.hint"))
+      #end
+      &lt;span class="xHint">$escapetool.xml($hint)&lt;/span>
+    &lt;/dt>
+    #set ($displayOutput = $lightboxConfigDoc.display($propertyName, $mode))
+    &lt;dd>#stripHTMLMacro($displayOutput)&lt;/dd>
```

Comment:
At first sight, this looks weird and like a hack.

Reply:
When an xproperty displayer produces HTML (like in edit mode) ``doc.display()`` wraps the output in ``{{html clean="false" wiki="false"}}`` which forces you to use wiki syntax (i.e. ``{{html wiki="true"}}``). This is very annoying. I try to avoid mixing HTML with wiki syntax (i.e. ``{{html wiki="true"}}``) as much as possible, so I prefer to "unwrap" the "HTML" returned by ``doc.display()`` so that I can use it directly in ``{{html clean="false"}}`` (no wiki syntax).

A comment explaining the need for this would not hurt of course.